mappers/refund.go

```diff
@@ -0,0 +1,35 @@
+package mappers
+
+import "github.com/companieshouse/payments.api.ch.gov.uk/models"
+
+func MapToRefundRest(response models.CreateRefundGovPayResponse) models.RefundResourceRest {
+	return models.RefundResourceRest{
+		RefundId:  response.RefundId,
+		CreatedAt: response.CreatedDate,
+		Amount:    response.Amount,
+		Links:     mapToRefundsLinksRest(response.Links),
+		Status:    response.Status,
+	}
+}
+
+func MapToRefundResponse(gpResponse models.CreateRefundGovPayResponse) models.CreateRefundResponse {
+	return models.CreateRefundResponse{
+		RefundId:    gpResponse.RefundId,
+		Amount:      gpResponse.Amount,
+		CreatedDate: gpResponse.CreatedDate,
+		Status:      gpResponse.Status,
+	}
+}
+
+func mapToRefundsLinksRest(db models.GovPayRefundLinks) models.RefundLinksRest {
+	return models.RefundLinksRest{
+		Self: models.RefundSelfRest{
+			HREF:   db.Self.HREF,
+			Method: db.Self.Method,
+		},
+		Payment: models.RefundPaymentRest{
+			HREF:   db.Payment.HREF,
+			Method: db.Payment.Method,
+		},
```

Comment:
can you ensure that none of the links that you are saving to the DB are Gov Pay links? Just looking at the existing Payment document in Mongo and in only includes links to CH APIs, not Gov Pay.

Reply:
@dalebradley , @divya051988 can you confirm we should only save non-govPay links?

Reply:
We save govPay links in `external_payment_status_url`

Reply:
Considering that we already have the payment link in the external_payment_status_url field, should I store the external refund link (the one that is stored in refund Self.HREF field) in `external_refund_url` field in the refund object?

Would that be acceptable?

Reply:
I don't see why not . Is there a specification where this is defined ?

Reply:
Added swagger specs for refunds, should be merged shortly.